packages/server/controllers/ThemeController.ts

```diff
@@ -0,0 +1,47 @@
+import type { Request, Response } from 'express'
+import { Theme } from '../models/Theme'
+
+export const getTheme = async (req: Request, res: Response) => {
+  try {
+    const userId = req.params.userId
+
+    if (!userId) {
+      res.status(404).send('User id not found')
+      return
+    }
+
+    const theme = await Theme.findOne({
+      where: {
+        userId: userId,
```

Comment:
[Предложение]
Можно оставить только `userId`, если ключ и значение в объекте совпадают
```suggestion
        userId,
```

Reply:
fix